src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,28 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, onTabSelected, selectedTab }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
```

Comment:
Also, you can use the destruction for `tab`

Reply:
I don't use destruction in this case, because I need a whole tab object in onClick function. Can I use the whole object after destruction?